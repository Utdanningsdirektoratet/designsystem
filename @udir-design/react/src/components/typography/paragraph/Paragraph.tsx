import { Paragraph } from '@digdir/designsystemet-react';

export { Paragraph };
export type { ParagraphProps } from '@digdir/designsystemet-react';
