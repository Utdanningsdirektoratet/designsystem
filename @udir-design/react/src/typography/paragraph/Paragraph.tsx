export { Paragraph, type ParagraphProps } from '@digdir/designsystemet-react';
