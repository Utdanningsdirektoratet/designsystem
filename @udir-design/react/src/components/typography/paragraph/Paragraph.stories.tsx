import type { Meta, StoryObj } from '@storybook/react';
import { Paragraph } from './Paragraph';

const meta: Meta<typeof Paragraph> = {
  component: Paragraph,
  tags: ['alpha'],
};

export default meta;
type Story = StoryObj<typeof Paragraph>;

export const Preview: Story = {
  args: {
    children: 'Paragraph',
  },
};
