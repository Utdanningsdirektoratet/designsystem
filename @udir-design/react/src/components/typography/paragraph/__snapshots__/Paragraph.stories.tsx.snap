// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Preview 1`] = `
<p data-variant="default">
  Paragraph
</p>
`;
